scipy/stats/_qmvnt.py

```diff
@@ -222,7 +216,7 @@ def _qmvn(m, covar, low, high, rng, lattice='cbc', n_batches=10):
         The low and high integration bounds.
     rng : Generator, optional
         default_rng(), yada, yada
-    lattice : 'cbc' or 'richtmyer' or callable
+    lattice : 'cbc' or callable
```

Comment:
This parameter is no longer used. It can be removed from the signature and docstring.